cmd/nodecmd/create.go

```diff
@@ -208,6 +220,16 @@ func createNodes(_ *cobra.Command, args []string) error {
 	if err = createClusterNodeConfig(network, cloudConfig, clusterName, cloudService); err != nil {
 		return err
 	}
+	monitoringInstanceNodeID := ""
+	monitoringInstancePublicIP := ""
+	if separateMonitoringInstance {
+		// take the last instance ID and public IP and assign them as those of the monitoring instance
+		monitoringInstanceNodeID = cloudConfig.InstanceIDs[len(cloudConfig.InstanceIDs)-1]
```

Comment:
the same issue. we should keep monitoring as separate entity in cloudConfig. this is very confusing